src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostOptions.cs

```diff
@@ -28,13 +26,12 @@ internal static class RemoteHostOptions
             storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + nameof(SolutionChecksumMonitorBackOffTimeSpanInMS)));
 
         // use 64bit OOP
-        public static readonly Option<bool> OOP64Bit = new Option<bool>(
-            nameof(InternalFeatureOnOffOptions), nameof(OOP64Bit), defaultValue: false,
+        public static readonly Option2<bool> OOP64Bit = new Option2<bool>(
+            nameof(InternalFeatureOnOffOptions), nameof(OOP64Bit), defaultValue: true,
             storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + nameof(OOP64Bit)));
 
         public static bool IsServiceHubProcess64Bit(HostWorkspaceServices services)
```

Comment:
>IsServiceHubProcess64Bit [](start = 27, length = 24)

`ServiceHubRemoteHostClient.RequestServiceAsync` uses `IsServiceHubProcess64Bit` to determine whether to use 32-bit or 64-bit ServiceHub configuration. That's not what we want to do anymore.
We should remove the 32-bit ServiceHub configurations.

Reply:
> We should remove the 32-bit ServiceHub configurations.

I plan to remove all the configuration files in a follow-up PR.

Reply:
I removed the Roslyn configuration files.

Reply:
➡️ The 32-bit functionality is now back in this pull request, with follow-up PR #46741 submitted to remove it later.